Art/mitul-bhatia-keyboard-animation/styles.css

```diff
@@ -0,0 +1,108 @@
+.macbook {
+    position: relative;
+    width: 500px;
+    height: 300px;
+    background-color: #333;
+    border-radius: 20px;
+    box-shadow: 0 10px 20px rgba(0, 0, 0, 0.3);
+    animation: macbook-rotate 10s linear infinite;
+  }
+  
+  .screen {
+    position: absolute;
+    top: 20px;
+    left: 20px;
+    width: 460px;
+    height: 290px;
+    background-color: #fff;
+    border-radius: 10px;
+    overflow: hidden;
+    animation: screen-open 1s ease-in-out;
+  }
+  
+  .screen-content {
+    width: 100%;
+    height: 100%;
+    background-image: url('https://via.placeholder.com/460x290');
```

Comment:
# Invalid URL ? 
When navigating to this url, we end up to a non working page.
Your url link is broken / please make sure to have a correct reachable image link